dist/module.mjs

```diff
@@ -41,8 +41,12 @@ const SBP_BASE_SELECTORS = {
   'sbp/selectors/register': function (sels) {
     const registered = []
     for (const selector in sels) {
-      const domain = domainFromSelector(selector)
-      if (selectors[selector]) {
+      const domainName = domainFromSelector(selector)
```

Comment:
This actually breaks line 59 above:

```js
if (selector === `${domain}/_init`) {
```

We both forgot to run `npm run flow` which caught this error.